Cargo.toml

```diff
@@ -9,9 +9,12 @@ repository = "https://github.com/wick3dr0se/dyhra"
 [dependencies]
 macroquad = "0.4.8"
 macroquad-tiled = "0.2.1"
-shipyard = "0.7.0"
+shipyard = { version = "0.7.0", default-features = false, features = [
+    "std",
+    "proc",
+] }
```

Comment:
This is unfortunate, it disables the `parallel` feature.
The scheduler is currently running systems on another thread even when there is only one system to run.
It shouldn't be too difficult to fix, I'll release a patch when it's fixed https://github.com/leudz/shipyard/issues/202.